packages/next-urql/src/__tests__/with-urql-client.spec.tsx

```diff
@@ -59,15 +59,16 @@ describe('withUrqlClient', () => {
       const tree = shallow(<Component {...props} />);
       const app = tree.find(MockApp);
 
-      expect(spyInitUrqlClient).toHaveBeenCalledTimes(2);
```

Comment:
This test description wasn't 💯 accurate. This test runs in a Node environment, so I'm not sure why I even included the "client-side in the component" piece. In any case, now that `urqlClient` is returned in `getInitialProps` on the server, `initUrqlClient` should only be called once on the server 👍